PilnaKalade/Assets/Scripts/Cards/PlayerDeck.cs

```diff
@@ -0,0 +1,30 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerDeck : MonoBehaviour
+{
+    private static List<CardData> deck = new List<CardData>();
```

Comment:
What if there were two PlayerDeck instances? (since PlayerDeck is not static, it looks like this could be the case). I don't think 2 players should share the same deck so either change PlayerDeck to static or make this member non-static